assets/js/easyspa.js

```diff
@@ -9,6 +9,7 @@
  * $('div').easySPALoader()
  */
 
+// noinspection BadExpressionStatementJS
```

Comment:
?

Reply:
https://github.com/irnc/javascript-practices/blob/master/BadExpressionStatementJS.md

Reply:
?

Reply:
Supposing its something meant for IDEs

Reply:
I use phpStorm, maybe this comment was "injected" by IDE.